backend/apid/actions/silenced.go

```diff
@@ -217,6 +218,11 @@ func (a SilencedController) Destroy(ctx context.Context, params QueryParams) err
 		}
 	}
 
+	// Unescape the ID parameter, in case we have something like an asterisk (*)
+	if unescapedID, err := url.QueryUnescape(id); err == nil {
```

Comment:
There's a bit of a discrepancy in the destroy function here. In most other controller methods we pass in the ID after calling `url.PathUnescape`.

I think that `PathUnescape` is correct here, not `QueryUnescape`, since the ID component is coming from the path. Since we are doing the rest of the unescaping in the router methods, we should do the escaping for destroy there as well.